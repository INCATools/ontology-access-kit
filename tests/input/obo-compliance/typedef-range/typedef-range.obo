format-version: 1.4
ontology: typedef-range

[Typedef]
id: R:1
range: Y:1

