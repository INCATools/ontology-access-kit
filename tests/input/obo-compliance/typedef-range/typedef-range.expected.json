{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/typedef-range.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ],
    "domainRangeAxioms" : [ {
      "predicateId" : "http://purl.obolibrary.org/obo/R_1",
      "rangeClassIds" : [ "http://purl.obolibrary.org/obo/Y_1" ]
    } ]
  } ]
}